mrblib/haconiwa/logger.rb

```diff
@@ -1,59 +1,66 @@
 module Haconiwa
   module Logger
+    DEBUG   = 0
+    NOTICE  = 1
+    INFO    = 2
+    WARNING = 3
+    ERROR   = 4
+
     extend self
     def setup(base)
       if Syslog.opened?
         Syslog.close
       end
       Syslog.open("haconiwa.#{base.name}")
+      @log_level = base.log_level
```

Comment:
This can suppress all of messages if `@log_level` is set to number larger than **5** . Is it intended? (I think it is not desirable)